app/javascript/packs/details_pg.js

```diff
@@ -0,0 +1,14 @@
+if (document.readyState !== "loading") {
+    onReady();
+} else {
+    document.addEventListener("DOMContentLoaded", onReady);
+}
+
+function onReady() {
```

Comment:
It looks like the indentation in this file is still 4 spaces when it should be 2.